src/main/java/org/jala/efeeder/details/DetailsCommand.java

```diff
@@ -216,4 +219,17 @@ private Timestamp getPaymentTime(Connection connection, String idFoodMeeting) th
 		FoodMeetingManager foodMeetingManager = new FoodMeetingManager(connection);
 		return foodMeetingManager.getFoodMeetingById(Integer.parseInt(idFoodMeeting)).getPaymentDate();
 	}
+	
+	private List<User> getUsersWithOrders(Connection connection, int idFoodMeeting) throws SQLException {
+		UserManager userManager = new UserManager(connection);
+		return userManager.getUsersWithOrders(idFoodMeeting);
+	}
+	
+	private double getSharedPriceByUser(double itemTotalPrice, int usersSize) {
+		double result = 0.0;
+		
+		result = itemTotalPrice > 0 ? (usersSize > 0 ? itemTotalPrice / usersSize : 0) : 0;
```

Comment:
there are two conditional in one single line, please use 'if' statement in order to get the code more readable